test/algorithm/std_fill.cpp

```diff
@@ -5,6 +5,11 @@
 // See accompanying file LICENSE_1_0.txt or copy at
 // http://www.boost.org/LICENSE_1_0.txt
 //
+#ifdef BOOST_GIL_USE_CONCEPT_CHECK
+// FIXME: Range as pixel does not seem to fulfil pixel concepts due to no specializations required:
```

Comment:
s/fulfil/fullfill/ :-)

Reply:
Hmm, https://tfd.com/fulfil

I'm British English biased though :)

Reply:
Hmm, ok, point taken. :-)

Reply:
Lets go for a compromise, https://en.oxforddictionaries.com/definition/fulfill :)